rocketchat_API/APISections/base.py

```diff
@@ -114,6 +114,40 @@ def call_api_post(self, method, files=None, use_json=None, **kwargs):
             timeout=self.timeout,
         )
 
+    def call_api_put(self, method, files=None, use_json=None, **kwargs):
+        reduced_args = self.__reduce_kwargs(kwargs)
+        # Since pass is a reserved word in Python it has to be injected on the request dict
+        # Some methods use pass (users.register) and others password (users.create)
+        if "password" in reduced_args and method != "users.create":
+            reduced_args["pass"] = reduced_args["password"]
+            del reduced_args["password"]
```

Comment:
Maybe let's get rid of this. It's not used and not tested. Let's introduce it when needed if needed.